src/function.rs

```diff
@@ -59,6 +60,17 @@ impl Function {
   }
 }
 
+fn abs_path(ctx: &FunctionContext, path: &str) -> Result<String, String> {
```

Comment:
I think `absolute_path` is a little clearer, and the context should be named `context` to match the other functions.

```suggestion
fn absolute_path(context: &FunctionContext, path: &str) -> Result<String, String> {
```

Reply:
To match the other functions.